crates/holochain/tests/authored_test/mod.rs

```diff
@@ -49,7 +49,7 @@ async fn authored_test() {
         .get_cell_triggers(&alice_call_data.cell_id)
         .await
         .unwrap();
-    triggers.publish_dht_ops.trigger(&"");
+    triggers.integrate_dht_ops.trigger(&"authored_test");
```

Comment:
Naughty test, relying on publish kicking integration. There's nothing to sys_validate because the DHT database already contained validated ops which were inserted locally. Trigger the integration workflow instead and this test works nicely again.